src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js

```diff
@@ -112,11 +110,13 @@ export class CernerCallToAction extends Component {
       });
       return (
         <div data-testid="cerner-cta-widget">
-          <AlertBox
-            headline="Something went wrong"
-            content="We’re sorry. Something went wrong on our end. Please try again later."
-            status="error"
-          />
+          <va-alert status="error">
+            <h3 slot="headline">"Something went wrong"</h3>
```

Comment:
`Something went wrong` probably should not have quotes.

Reply:
good catch, ty